### PR TITLE
feat: Support form to include session replay and admin links

### PR DESCRIPTION
**Pull request by tiina303**
## Problem

Follow-up for https://github.com/PostHog/posthog/pull/15018

## Changes

Added links to Zendesk ticket for quick access to the recording and admin panel 
Example ticket: https://posthoghelp.zendesk.com/agent/tickets/2456 (moved to PostHog: https://us.posthog.com/project/2/support/tickets/5417)

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/api/user.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ class UserSerializer(serializers.ModelSerializer):
     is_impersonated = serializers.SerializerMethodField()
     is_2fa_enabled = serializers.SerializerMethodField()
     has_social_auth = serializers.SerializerMethodField()
+    postgres_id = serializers.SerializerMethodField()
```
Comment (benjackwhite) [inline posthog/api/user.py:62]: I don't think we should expose this. Instead we can just use the email and link to the list view like:

https://app.posthog.com/admin/posthog/user/?q=ben%40posthog.com

**Review by benjackwhite — CHANGES_REQUESTED** [review]
I like it except for exposing the postgres_id

File: posthog/api/user.py (hunk 1 above)
Comment (tiina303) [reply posthog/api/user.py:62]: hmm... this reminds me I need to also call out the current project name as the user might have changed their organization and project by this point.

**Review by benjackwhite — APPROVED** [review]

File: frontend/src/lib/components/Support/supportLogic.ts [hunk 2]
```diff
@@ -58,7 +78,9 @@ export const supportLogic = kea<supportLogicType>([
                             `\n\n-----` +
                             `\nKind: ${kind}` +
                             `\nTarget area: ${target_area}` +
-                            `\nInternal link: http://go/ticketByUUID/${zendesk_ticket_uuid}`,
+                            `\nInternal link: http://go/ticketByUUID/${zendesk_ticket_uuid}` +
```
Comment (lharries) [inline frontend/src/lib/components/Support/supportLogic.ts:81]: I think we need spaces between the links

**Pull request merged**